appmap/test/test_detect_enabled.py

```diff
@@ -0,0 +1,143 @@
+"""
+Test detecting if AppMap is enabled
+"""
+import os
+import pytest
+import unittest
+from unittest.mock import patch
+from appmap._implementation.detect_enabled import DetectEnabled, RECORDING_METHODS
+from appmap._implementation.testing_framework import file_write, file_delete
+
+class TestDetectEnabled:
+    @staticmethod
+    def setup_method():
+        DetectEnabled.clear_cache()
+
+    @patch.dict(os.environ, {"APPMAP": "false"})
+    def test_none__appmap_disabled(self):
```

Comment:
I think it would be better to use classes to scope test cases, rather than a naming convention.

Reply:
Hmmm. You marked this as resolved, but didn't change the naming convention? I guess that's your call, but maybe you could explain?